plugins/pipelines/sync.js

```diff
@@ -42,39 +42,41 @@ module.exports = () => ({
                 throw boom.notFound(`User ${username} does not exist`);
             }
 
-            // Use parent's scmUri if pipeline is child pipeline and using read-only SCM
-            const scmUri = await getScmUri({ pipeline, pipelineFactory });
+            if (!scope.includes('admin')) {
```

Comment:
what if the user is an admin and also has write access on the repo ?

Reply:
refactored